test-prow-e2e.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+set -eExuo pipefail
+
+NAMESPACE=""            # TODO
```

Comment:
@naveenpaul1 Just confirming here that the decided namespace is `data-federation`, right?